cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -34,11 +34,13 @@ import com.github.ajalt.clikt.parameters.options.split
 import com.github.ajalt.clikt.parameters.options.splitPair
 import com.github.ajalt.clikt.parameters.types.file
 
-import java.io.File
-
-import kotlin.time.TimedValue
 import kotlin.time.measureTimedValue
 
+import kotlinx.coroutines.Dispatchers
```

Comment:
Commit: Is there a specific performance issue which is the reason for this change?

Reply:
I'm surprised that you're asking, because as you know there is https://github.com/oss-review-toolkit/ort/issues/3511. While the root cause of the performance issues might be addressed differently, running reporters in parallel certainly helps.

Reply:
Here are some more numbers:

```
Successfully created the 'AsciiDocTemplate' report at [/output/results/AsciiDoc_disclosure_document.pdf, /output/results/AsciiDoc_vulnerability_report.pdf] in 254.852095529s.
Successfully created the 'CycloneDx' report at [/output/results/CycloneDX-BOM.xml] in 35.434101825s.
Successfully created the 'NoticeTemplate' report at [/output/results/NOTICE_default, /output/results/NOTICE_summary] in 38.216188753s.
Successfully created the 'SpdxDocument' report at [/output/results/document.spdx.yml] in 0.724275908s.
Successfully created the 'StaticHtml' report at [/output/results/scan-report.html] in 1.275565648s.
Successfully created the 'WebApp' report at [/output/results/scan-report-web-app.html] in 14.572000009s.
```

With this change, the total time is only slightly more than the longest time of a single report generation. That is, in this example we safe roughly 1m30s of time.

Reply:
> I'm surprised that you're asking, because as you know there is #3511. While the root cause of the performance issues might be addressed differently, running reporters in parallel certainly helps.

I'm asking this "because" I'm actually looking into #3511 and as I know the root cause. I wondered if we parallelize here just because we can - or if there is a specific reason. In particular I wondered if this is parallelization is needed after #3511 is fixed.

Doing things faster definetly makes sense - in context of this change I'm not certain whether or how it affects the peaks in memory consumption. Have you considered that?

Reply:
> in context of this change I'm not certain whether or how it affects the peaks in memory consumption.

I'm not certain either, but I'm willing to find out empirically. If it turns out that peak memory consumption now becomes an issue here, even after your upcoming changes, then we could make parallelism optional here easily.